src/comm_droplet.F

```diff
@@ -649,9 +649,9 @@ subroutine setupIndexPointers(Cnt,ptrCnt)
 
       ptrCnt(inorth) = 1
       ptrCnt(isouth) = ptrCnt(inorth) + Cnt(inorth)
-      ptrCnt(ieast)  = ptrCnt(isouth) + Cnt(isouth)
-      ptrCnt(iwest)  = ptrCnt(ieast)  + Cnt(ieast)
-      ptrCnt(inw)    = ptrCnt(iwest)  + Cnt(iwest)
+      ptrCnt(iwest)  = ptrCnt(isouth) + Cnt(isouth)
+      ptrCnt(ieast)  = ptrCnt(iwest)  + Cnt(iwest)
+      ptrCnt(inw)    = ptrCnt(ieast)  + Cnt(ieast)
```

Comment:
is there a particular reason to switch the order here?

Reply:
The order of the droplets in the Depart_ind was not consistent with the order of the integer direction pointers (inorth,isouth, iwest,ieast,...)  Making it consistent fixed a very vexing correctness bug.

Reply:
Thanks.